tools/http_server.py

```diff
@@ -50,9 +51,11 @@ def spawn():
     r_thread = Thread(target=rs.serve_forever)
     r_thread.daemon = True
     r_thread.start()
-    sleep(1)  # TODO I'm too lazy to figure out how to do this properly.
```

Comment:
don’t remove the sleep here - this function is called elsewhere and it needs it.